migrations/127_simple_quests.js

```diff
@@ -0,0 +1,57 @@
+const {deployProxy, upgradeProxy} = require('@openzeppelin/truffle-upgrades');
+
+const SimpleQuests = artifacts.require("SimpleQuests");
+const Characters = artifacts.require("Characters");
+const Weapons = artifacts.require("Weapons");
+const Junk = artifacts.require("Junk");
+const RaidTrinket = artifacts.require("RaidTrinket");
+const Shields = artifacts.require("Shields");
+const BurningManager = artifacts.require("BurningManager");
+const SafeRandoms = artifacts.require("SafeRandoms");
+const Raid1 = artifacts.require("Raid1");
+
+module.exports = async function (deployer, network, accounts) {
+  const characters = await upgradeProxy(Characters.address, Characters, {deployer});
```

Comment:
If we can be certain the frontend flag won't break on mainnet with the contract not being deployed, we could save a bit of money now by adding an if branch to only deploy for testnets. Assuming we will have to completely redeploy later with the generalized interface

Reply:
We do have a frontend flag, ok, I'll add the if here